### PR TITLE
update release handbook

### PR DESCRIPTION
**Pull request by klumb**
Signed-off-by: Kevin Lumbard <kmlumbard@gmail.com>

Added detail to the release process. Not completely sure on all of this so I welcome discussion.

### CODE REVIEW

**Comment by klumb** [comment]
This metric is part of the continuous release process and the information displayed on the website may still be edited prior to the regular release. - Matt

**Comment by klumb** [comment]
continuous release list in release notes

**Comment by klumb** [comment]
no labels on metric release page

**Comment by klumb** [comment]
This metric is part of the continuous release process and has passed the 30 day comment period. This metric could, possibly, be edited prior to the regular release.

File: community-handbook/metrics-release.md [hunk 1]
```diff
@@ -2,27 +2,35 @@
...
-3. Include inside the metric that it will be included in next release.
-4. Create an issue for collecting feedback before release.
-5. Announce a 1-month feedback window on the mailing list and point to the issue.
-6. Address feedback.
-7. Update the website to include the new metric.
-8. Update the release notes to indicate that we changed the release.
+4. Remove the “will be included in next release” label on new metrics markdown pages and on the website.
+5. Create a GitHub release label (process is similar to creating a release branch) for all metrics that completed the “Continuous Metric Contribution”. This release label will replace the working markdown page the website was using during the continuous release period, creating a static version of the metric. The website will be edited to point to this static markdown page.
+6. The release notes will be updated - based on each of the working groups release issues - on the website to state that all changes are part of the new release. 
+7. The release gets a new version number.
```
Comment (GeorgLink) [inline community-handbook/metrics-release.md:35]: Item 7 should move up before item 5. We need the version number for the tagging.

**Comment by GeorgLink** [comment]
Disclaimer shared with the Common WG: This metric is part of the continuous release process, has been through the 30-day review period, and the information displayed on the website may still be edited prior to the regular release

**Comment by GeorgLink** [comment]
@klumb where are we on this PR? I am referencing the community handbook in the CHAOSS weekly and it would be great to get the update merged.

**Comment by klumb** [comment]
@GeorgLink I accepted your review changes. I would be comfortable merging this.

**Comment by klumb** [comment]
However for some reason it DCO is failing now.

**Comment by GeorgLink** [comment]
That's unfortunate :(

**Comment by vchrombie** [comment]
Hi @klumb 
I am really not sure about this problem, but I think I have a workaround for this.
Maybe you can try on one commit and see if this thing works. 
You can revert back the change if it doesn't.

I think that the sign-off is missing in the last two commits. Try adding your sign-off.
```
Co-Authored-By: Georg J.P. Link <linkgeorg@gmail.com>
Signed-off-by: Kevin Lumbard <kmlumbard@gmail.com>
```

You need to edit the existing commit message. Just try adding this to the last commit and check if it is working. You can do this by
```
$ git commit --ammend
```
This should open an editor for editing the commit message. Add your sign-off and force push.
```
$ git push -f origin branch_name
```
I hope it helps.

**Comment by vchrombie** [comment]
Hi @klumb 

I checked the error log of DCO. You can also do the same by clicking the `Details` beside the DCO check in this PR.

I was correct, the sign-off was missing in the last two commits which led to the failure.
I have found another workaround from the DCO help.

```
git rebase HEAD~3 --signoff
git push --force-with-lease origin master
```

I hope this helps. Please let me know if you have any questions or trouble.

**Comment by klumb** [comment]
Would anyone mind if I set the DCO to pass? I am sure it is from Georg's commits.

**Comment by GeorgLink** [comment]
no, I don't mind. We can squash merge and then it will not show up in the git log

**Pull request merged**